### DISCUSSION TITLE
Shipyard price multiplier in campaign settings does not affect the cost of installing a new turret

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#13981] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![602960_20240520185750_1](https://github.com/FakeFishGames/Barotrauma/assets/47554589/5e52f7a0-8eb1-4095-8bb4-67cf60c00f0f)
![602960_20240520185800_1](https://github.com/FakeFishGames/Barotrauma/assets/47554589/9aeea67d-4af2-44da-af85-f3a293752dba)

### Reproduction steps

1. Set the shop and shipyard price multipliers to 0.1.
2. Start the campaign on a submarine with an empty hardpoint.
3. Check the shipyard and compare the turret prices to the prices of all other upgrades.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by DrBruhman** [comment]
same as wending machines

**Reply by Regalis11** [reply]
In my opinion, "shipyard price multiplier" should not affect vending machines. Shipyard purchases in this context means purchases for repairs/upgrades and such.

**Reply by NotWendy** [reply]
Maybe they meant the regular shop price multiplier not affecting vending machines. Which is also true!

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/d9d443b44ece6fa4730e2309805c39464be85799

**Comment by Jasontti** [comment]
Tested the fix for shipyard price multiplier and found no issues.
Closing this discussion as resolved.
